### DISCUSSION TITLE
[추현규] 과제 1번 제출합니다.

### DISCUSSION THREAD

**Discussion by choo043** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#24] [2024년도 과제 제출]
![image](https://github.com/SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course/assets/76640650/49a7ae5b-8d01-46b2-9e60-c00433d05563)

**Comment by Jinsun-Lee** [comment]
안녕하세요.
자율주행 통합실습 담당 조교입니다. 

아래 링크에서 교육 수강 전 공지사항을 확인해 주시길 바랍니다.
https://github.com/SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course/discussions/15

감사합니다.

**Comment by choo043** [comment]
감사합니다!! 오늘도 좋은 하루 보내시길 바랍니다!


-----Original Message-----
